lib/jsdom/living/nodes/HTMLSelectElement-impl.js

```diff
@@ -82,19 +83,69 @@ class HTMLSelectElementImpl extends HTMLElementImpl {
       selected.forEach((option, index) => {
         option._selectedness = index === selected.length - 1;
       });
+      this._selectedOption = selected[selected.length - 1];
```

Comment:
Shouldn't you also update this around line 77?

Reply:
I didn't think was necessary due to mostly going via fast path when selectness is not set on incoming child but I think its safer to do so will add.